modules/v2x/v2x_proxy/app/v2x_proxy.cc

```diff
@@ -57,7 +57,8 @@ V2xProxy::V2xProxy()
   }
 
   hdmap_.reset(new apollo::hdmap::HDMap());
-  std::string map_name = FLAGS_map_dir + "/" + FLAGS_base_map_filename;
+  std::string map_name = absl::GetFlag(FLAGS_map_dir) + "/" +
+                         absl::GetFlag(FLAGS_base_map_filename);
```

Comment:
Use `absl::StrCat` for all such cases.
https://abseil.io/tips/3

Reply:
ok